src/vtp/ops/verify_ballot_receipt_operation.py

```diff
@@ -299,31 +282,26 @@ def vet_a_row():
                 vet_a_row()
 
         # Summerize
+        self.imprimir("############")
         if error_digests:
-            logging.error(
-                "############\n"
+            self.imprimir(
                 "[ERROR]: ballot receipt INVALID - the supplied ballot receipt has "
-                "%s errors.\n############",
-                len(error_digests),
+                "{len(error_digests)} errors."
             )
         else:
-            print(
-                "############\n"
-                "[GOOD]: ballot receipt VALID - no digest errors found\n############"
-            )
+            self.imprimir("[GOOD]: ballot receipt VALID - no digest errors found")
+        self.imprimir("############")
 
     # pylint: disable=duplicate-code
     def run(
         self,
         receipt_file: str = "",
+        receipt_data: list = None,
```

Comment:
```suggestion
        receipt_data: list[str] = None,
```

Reply:
@ion-oset - same for these as well - should they really be list[list[str]]?

Reply:
Yes.